packages/react/src/Card/Card.module.css

```
@@ -221,6 +221,10 @@
   z-index: -1;
 }
 
+.Card--expandableArrow {
+  margin-left: var(--base-size-4);
```

Since the gap between the label and the arrow plus external icon in `Link` is now managed at the parent level (i.e., `.Link`), and since `Card` links do not use those parent styles, we need to reintroduce the spacing here.

---

Sounds good @danielguillan. Nit but could we use a logical property here instead?

```suggestion
  margin-inline-start: var(--base-size-4);
```